src/app/admin/events/EventsForm.tsx

```diff
@@ -55,10 +55,10 @@ export default function EventsForm() {
 			title_sv: "",
 			title_en: "",
 			council_id: 0,
-			starts_at: "2025-01-31T18:00",
-			ends_at: "2025-01-31T20:00",
-			signup_start: "2025-01-31T18:00",
-			signup_end: "2025-01-31T20:00",
+			starts_at: new Date().toLocaleString(),
+			ends_at: new Date().toLocaleString(),
+			signup_start: new Date().toLocaleString(),
+			signup_end: new Date().toLocaleString(),
```

Comment:
I'm sorry for the confusion, but `Date.toLocaleString` is best used to display dates to the user (an example output is "2/10/2025, 3:07:28 PM", which is very pretty to look at but as pleasant to parse on the backend). So, wherever `Date`s have to be transmitted as strings, you should keep using `Date.toISOString`.